projects/github-actions/repo-gardening/src/tasks/triage-new-issues/index.js

```diff
@@ -148,6 +206,40 @@ async function triageNewIssues( payload, octokit ) {
 			issue_number: number,
 			labels: [ `[Pri] ${ priority }` ],
 		} );
+	} else if ( priority === null && ! hasPriorityLabel ) {
+		// No priority found, and no priority label.
+		// Let's notify the team so they can prioritize the issue appropriately.
+		// So far only enabled in the Calypso repo.
+		if ( 'Automattic/wp-calypso' !== repository.full_name ) {
+			return;
+		}
+
+		// No Slack tokens, we won't be able to escalate. Bail.
+		const slackToken = getInput( 'slack_token' );
+		const channel = getInput( 'slack_quality_channel' );
+		if ( ! slackToken || ! channel ) {
+			debug(
+				`triage-new-issues: No Slack token or channel found. Not sending any Slack notification for #${ number }.`
+			);
+			return null;
+		}
+
+		// Check if Kitkat input was already requested for that issue.
+		const hasBeenRequested = await hasKitkatSignalLabel( octokit, ownerLogin, name, number );
+		if ( hasBeenRequested ) {
+			debug(
+				`triage-new-issues: Kitkat input was already requested for issue #${ number }. Aborting.`
+			);
+			return;
+		}
+
+		debug(
+			`triage-new-issues: #${ number } doesn't have a Priority set. Sending in Slack message to the Kitkat team.`
+		);
+
+		const message = '@kitkat-team New bug missing priority. Please do a priority assessment.';
+		const slackMessageFormat = formatSlackMessage( payload, channel, message );
+		await sendSlackMessage( message, channel, slackToken, payload, slackMessageFormat );
```

Comment:
Shouldn't something in here be actually adding that `[Pri] TBD` label to the issue?

Reply:
Yeah, my bad I had forgotten to push that. 54d2af9c96b5bcafbc623a13b5a392bb7028f168